hazelcast/src/main/java/com/hazelcast/spi/discovery/multicast/MulticastProperties.java

```diff
@@ -39,6 +40,22 @@
      */
     public static final PropertyDefinition GROUP = property("group", STRING);
 
+    /**
+     * Property which determines if Java Serialization is used ({@code false}) or rather the safer and portable one
+     * ({@code true}).
+     * <p/>
+     * Safe serialization format of the MulticastMemberInfo:
+     *
+     * <pre>
+     * boolean: flag if memberInfo provided (false mean memberInfo is null)
+     * UTF-8:   host
+     * int:     port
+     * </pre>
+     *
+     * @since 5.1
+     */
+    public static final PropertyDefinition SAFE_SERIALIZATION = property("safe-serialization", BOOLEAN);
```

Comment:
can we have a concise description of what we mean by safe-serialization here as javaDoc? Better if we include an example breach scenario in it.

Reply:
I will extend the description and add a link to an OWASP page related to the deserialization.
I don't want to share exactly the attack vector as we don't do that for found vulnerabilities.